src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/LogMailSender.java

```diff
@@ -8,8 +9,21 @@
 @Profile({"dev", "test"})
 @Slf4j
 public class LogMailSender implements MailSender {
+
+    private final MailPostAction mailPostAction;
+
+    @Autowired
+    public LogMailSender(MailPostAction mailPostAction) {
+        this.mailPostAction = mailPostAction;
+    }
+
     @Override
     public void sendEmail(String recipientAddress, String subject, String htmlContent) {
         log.debug("Sending email to '{}'\nSubject: {}\n{}", recipientAddress, subject, htmlContent);
+        mailPostAction.postAction();
+    }
+
+    public MailPostAction getMailPostAction() {
```

Comment:
Can be removed and replaced with Lombok's `@Getter` annotation on `mailPostAction` field declaration.